pkgs/development/tools/continuous-integration/buildbot/master.nix

```
@@ -70,14 +70,14 @@ let
 
   package = buildPythonApplication rec {
     pname = "buildbot";
-    version = "3.11.0";
+    version = "3.11.1";
     format = "pyproject";
```

For the next PR:
```suggestion
    pyproject = true;
```
The same should also be done for all plugins.